it/controllers/SelfEmployedCYAControllerISpec.scala

```diff
@@ -134,6 +143,7 @@ class SelfEmployedCYAControllerISpec  extends ComponentSpecBase with FeatureSwit
       stubAuthSuccess()
       stubGetSelfEmployments(businessAccountingMethodKey)(OK, Json.toJson(testAccountingMethodModel))
       stubGetSelfEmployments(businessesKey)(OK, Json.toJson(testBusinesses))
+      stubSaveSelfEmployments(businessesKey, Json.toJson(testConfirmedBusinesses))(OK)
       And("save & retrieve feature switch is enabled")
       enable(SaveAndRetrieve)
 
```

Comment:
hi I know you did not write this part of the code, but can you help to amend 

in row 153, can you amend the description from "Should return a SEE_OTHER with a redirect location of confirmation" to "Should return a SEE_OTHER with a redirect location of task list page"

in row 171, do the same change the description to 
"Should return a SEE_OTHER with a redirect location of self-employed CYA page"